msteams-platform/concepts/design/map-use-cases.md

```diff
@@ -43,30 +72,34 @@ Send asynchronous notifications and alerts to your users in Teams. Use interacti
 
 * Conversational bots - Send proactive messages to groups, channels, or individual users.
 
-* Connectors & incoming webhooks - Allow a channel to subscribe to receive messages. With a connector let users tailor the subscription with a configuration page.
+* Connectors and incoming webhooks - Permit a channel to subscribe to receive messages. A connector lets users tailor the subscription with a configuration page.
 
 ## Ask questions and get answers
 
-People have questions. You've probably got a lot of the answers stored away somewhere. Unfortunately, its often quite difficult to connect the two together.
+People have questions and you probably got a lot of the answers stored away somewhere. Unfortunately, it's often quite difficult to connect the two.
 
-* Conversational bots - Natural language processing, AI, machine learning, all the buzzwords. Use a bot powered by the intelligent cloud to connect your users to the answers they need.
+* Conversational bots - Natural language processing, AI, Machine Learning (ML), all the buzzwords. Use a bot powered by the intelligent cloud to connect your users to the answers they need.
```

Comment:
@KirtiPereira We don't need to capitalize "Machine Learning" or use an acronym here.